build.bat

```diff
@@ -93,21 +93,48 @@ if %build_rime% == 0 (
   set build_weasel=1
 )))))
 
+rem quit WeaselServer.exe before building
 cd /d %WEASEL_ROOT%
 if exist output\weaselserver.exe (
   output\weaselserver.exe /q
 )
 
+rem build booost
 if %build_boost% == 1 (
   call :build_boost
   if errorlevel 1 exit /b 1
   cd /d %WEASEL_ROOT%
 )
 
+rem build librime x64 and Win32
 if %build_rime% == 1 (
+  rem build 64 bit librime
   if not exist librime\build.bat (
     git submodule update --init --recursive
   )
+  set ARCH=x64
+  cd %WEASEL_ROOT%\librime
+  rem clean cache before building
+  if exist build ( del /S /Q build )
+  if exist dist ( del /S /Q dist )
+  if exist deps\glog\build ( del /S /Q deps\glog\build )
+  if exist deps\googletest\build ( del /S /Q deps\googletest\build )
+  if exist deps\leveldb\build ( del /S /Q deps\leveldb\build )
+  if exist deps\marisa-trie\build ( del /S /Q deps\marisa-trie\build )
+  if exist deps\opencc\build ( del /S /Q deps\opencc\build )
+  if exist deps\yaml-cpp\build ( del /S /Q deps\yaml-cpp\build )
+  if exist lib ( del /S /Q lib )
+
+  rem restore backuped x64 build

```

Comment:
changes not required now. but I want to explore better ways than backup/restore.

Reply:
> changes not required now. but I want to explore better ways than backup/restore.

this is mostly for reducing build-time local. and I found command `move` might cause problems, checking ...

Reply:
it's caused by preset ARCH in env template, which will stop the `set ARCH=x64` and `set ARCH=Win32` in build.bat

Reply:
it's for the case that previous Win32 build job failed, if not clean the caches, it will failed again.